### DISCUSSION TITLE
멀티플레이 테스트 오류관련해서 질문드립니다.

### DISCUSSION THREAD

**Discussion by heavybowgun** [naverz/zepeto-studio-kor#889] [5. Q&A]
멀티예제 샘플 통해서 작업을 진행하고있습니다.

멀티 예제 샘플 오브젝트들만 지우고 거기에 빌드잇에서 작업한걸 불러와 작업을 진행하고 멀티서버킨다음 스타트버튼을 눌러봤는데

오류가 나옵니다.

멀티쪽 ts나 관련 파일들은 하나도 안건드렸는데 오류나는 이유를 몰라 질문드립니다.

혹시나 멀티플레이 예제 샘플에 추가적인 작업을 해야 멀티서버가 구동되는건가요?

아니면 다른 오류인가요?

![오류1](https://user-images.githubusercontent.com/107227678/173797172-3ea17718-c063-4d56-ae2c-a2489a2c547b.JPG)
![오류2](https://user-images.githubusercontent.com/107227678/173797185-9ebfbb1c-fcc3-4312-8477-42b38c1649b7.JPG)

**Comment by YSHFM** [comment]
오류메시지에 써져있는대로 , js파일로 확인해보신다음 51행 체크해보셔야 할 것 같습니다. ts스크립트 선택하면 인스펙터에서 js파일 여는 옵션이 있습니다.

**Comment by hanaaple** [comment]
지금으로선 clientstarter를 봐야 알 것 같습니다
